flowermd/tests/base/test_system.py

```diff
@@ -201,6 +201,32 @@ def test_pack_box(self, benzene_molecule):
             high_density_system.box.lengths
         )
 
+    def test_pack_seed(self, benzene_molecule):
+        benzene_mol = benzene_molecule(n_mols=3)
+        default_seed = Pack(molecules=[benzene_mol], density=0.1)
+        change_seed = Pack(molecules=[benzene_mol], density=0.1, seed=12340)
+        assert np.prod(low_density_system.box.lengths) > np.prod(
```

Comment:
Here, instead of testing box lengths (which should be the same regardless of `seed`), let's test the positions of the system and make sure they are different.

Using something like [np.array](https://numpy.org/doc/2.1/reference/generated/numpy.array_equal.html)

Something along the lines of this:
```
assert not np.array_equal(system_one.xyz, sytem_two.xyz)
```